src/main/groovy/com/marklogic/gradle/task/admin/InitTask.groovy

```diff
@@ -39,6 +39,15 @@ class InitTask extends MarkLogicTask {
 		if (project.hasProperty("mlLicensee")) {
 			licensee = project.property("mlLicensee")
 		}
+
+		// Permit this task to run without having mlUsername and mlPassword defined
+		if (!project.hasProperty("mlUsername")) {
```

Comment:
I may have mis-spoken here - this looks like it should actually be fixed in ml-app-deployer - i.e. the `init` and `installAdmin` methods should be smart enough to not fail if there's no authentication.